streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopicManagerTest.java

```diff
@@ -84,6 +87,10 @@
 
     @Before
     public void init() {
+        // When executing on Jenkins, the thread name is set to an unknown value,
+        // hence, we need to set it explicitly to make our log-assertions pass
+        Thread.currentThread().setName(threadName);
```

Comment:
Hmm, this might be surprising. Won't this cause the test executor thread to be called "threadName" from now until the end of the build?

Do you think we could instead get the currentThread's name and use that in the assertions? Or otherwise make the assertions agnostic to the name of the thread?

Reply:
Getting the name is much better! Should have done this for the beginning on...